news/2024/2024-09-30-the-followpoint-d-i-o-the-charismatic-caster.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: "The Followpoint: D I O, the Charismatic Caster"
+date: 2024-09-30 17:00:00 +0000
+series: The Followpoint
+---
+
+...You expected a tournament player, but it was me, D I O!
```

Comment:
you'd quote this since it wouldn't be utiba's pov (the author of the article)

imo this would fit better as part of the banner graphic

```suggestion
"...You expected a tournament player, but it was me, D I O!"
```